tools/fastq_provider/fastq_provider.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="fastq_provider" name="Atlas FASTQ provider" version="@TOOL_VERSION@+galaxy" python_template_version="3.5" profile="20.01">
+    <description> Scripts and utilities for providing fastqs to workflows. </description>
```

Comment:
This is meant to be text that gets read right after the name, so thing of something that will read well after "Atlas FASTQ provider".

Reply:
thanks @pcm32  The current description comes from the source https://github.com/ebi-gene-expression-group/atlas-fastq-provider It will say now "Retrieval and download of FASTQ files from ENA and other repositories such as HCA or SRA. The tool is used in production pipelines of EMBL-EBI Expression Atlas and Single Cell Expression Atlas. "

Reply:
The SRA is not a source repository distinct from ENA for the purposes of this tool. It does however use SRA-type archives sometimes rather than pulling fastqs directly, which is probably the source of the confusion.

Reply:
edited, thanks @pinin4fjords